### PR TITLE
chore: greet contributor

### PR DESCRIPTION
**Pull request by OleksandrNechai**
Extending our contributor GHA. This time, let's greet contributor and let them know what to expect next. Examples of messages see below in PR :point_down:

### CODE REVIEW

File: .github/workflows/notify-jira-about-contribution.yml [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
   pull_request:
-    types: [opened]
+    # types: [opened]
     branches:
       - master
 
 jobs:
   notify-jira:
     # if external contribution
-    if: ${{!contains(fromJson('["denieler", "TomasSlama", "OleksandrNechai", "OndrejTuma", "dmaklygin1", "LashaJini", "toptal-build"]'), github.event.pull_request.user.login)}}
```
Comment (LashaJini) [inline .github/workflows/notify-jira-about-contribution.yml:13]: I guess we will be using this line of code a lot of times 😁. Is it possible to define it somewhere and have it as a single source of truth?

Comment (OleksandrNechai) [reply .github/workflows/notify-jira-about-contribution.yml:13]: I think it is possible. Do you want to figure out how? My building time on the calendar comes to the end :)

Comment (LashaJini) [reply .github/workflows/notify-jira-about-contribution.yml:13]: Sure, why not.

Comment (OleksandrNechai) [reply .github/workflows/notify-jira-about-contribution.yml:13]: Feel free to do it. But only after TextEditor styles are done :)

File: .github/workflows/handle-contribution.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Handle contribution
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened]
+    branches:
+      - master
```
Comment (LashaJini) [inline .github/workflows/handle-contribution.yml:8]: I was wondering, what will happen when someone opens PR on another branch, they might still need this greeting in that case also. wdyt? :thinking:

Comment (LashaJini) [reply .github/workflows/handle-contribution.yml:8]: No, they won't!

**Review by LashaJini — APPROVED** [review]

**Review by dmaklygin1 — APPROVED** [review]

**Pull request merged**